zc_events/responses.py

```diff
@@ -56,13 +56,13 @@ def __init__(self, response):
 def get_wrapped(func, wrapper_template, evaldict):
     # Preserve the argspec for the wrapped function so that testing
     # tools such as pytest can continue to use their fixture injection.
-    args, a, kw, defaults = inspect.getargspec(func)
+    args, a, kw, defaults = inspect.getargspec(func)  # pylint: disable=deprecated-method
 
-    signature = inspect.formatargspec(args, a, kw, defaults)
+    signature = inspect.formatargspec(args, a, kw, defaults)  # pylint: disable=deprecated-method
     is_bound_method = hasattr(func, '__self__')
     if is_bound_method:
         args = args[1:]     # Omit 'self'
-    callargs = inspect.formatargspec(args, a, kw, None)
+    callargs = inspect.formatargspec(args, a, kw, None)  # pylint: disable=deprecated-method
```

Comment:
`getargspec` and `formatargspec` are deprecated in python 3.5, didn't want to spend time figuring out the new replacements